docs/reference/glossary.md

```diff
@@ -181,22 +187,60 @@ labeled using [tags](#tag).
 Here is an example of the shared [nginx repository](https://hub.docker.com/_/nginx/)
 and its [tags](https://hub.docker.com/r/library/nginx/tags/)
 
-## Swarm
+## routing mesh
+
+The swarm mode ingress [routing mesh](../swarm/ingress.md) lets you expose ports
```

Comment:
We're defining the concept in a recursive way. Do we need to include "routing mesh" in the glossary? Do we use that term a lot?